app/src/main/java/com/arygm/quickfix/model/profile/WorkerProfileRepositoryFirestore.kt

```diff
@@ -143,46 +143,6 @@ class WorkerProfileRepositoryFirestore(private val db: FirebaseFirestore) : Prof
         }
   }
 
-  private fun String.toWorkerCategory(): WorkerCategory? {
-    return when (this) {
-      "ConstructionAndMaintenance.GeneralLaborer" ->
-          WorkerCategory.ConstructionAndMaintenance.GeneralLaborer
-      "ConstructionAndMaintenance.Mason" -> WorkerCategory.ConstructionAndMaintenance.Mason
-      "HomeImprovementAndRepair.Handyman" -> WorkerCategory.HomeImprovementAndRepair.Handyman
-      "HomeImprovementAndRepair.FlooringInstaller" ->
-          WorkerCategory.HomeImprovementAndRepair.FlooringInstaller
-      "MechanicalAndVehicleMaintenance.AutoMechanic" ->
-          WorkerCategory.MechanicalAndVehicleMaintenance.AutoMechanic
-      "MechanicalAndVehicleMaintenance.DieselMechanic" ->
-          WorkerCategory.MechanicalAndVehicleMaintenance.DieselMechanic
-      else -> null
-    }
-  }
-
-  private fun WorkerCategory.toFirestoreString(): String {
-    return when (this) {
-      is WorkerCategory.ConstructionAndMaintenance.GeneralLaborer ->
-          "ConstructionAndMaintenance.GeneralLaborer"
-      is WorkerCategory.ConstructionAndMaintenance.Mason -> "ConstructionAndMaintenance.Mason"
-      is WorkerCategory.HomeImprovementAndRepair.Handyman -> "HomeImprovementAndRepair.Handyman"
-      is WorkerCategory.HomeImprovementAndRepair.FlooringInstaller ->
-          "HomeImprovementAndRepair.FlooringInstaller"
-      is WorkerCategory.MechanicalAndVehicleMaintenance.AutoMechanic ->
-          "MechanicalAndVehicleMaintenance.AutoMechanic"
-      is WorkerCategory.MechanicalAndVehicleMaintenance.DieselMechanic ->
-          "MechanicalAndVehicleMaintenance.DieselMechanic"
-    }
```

Comment:
Are you sure this wont be needed for Workers metadata?

Reply:
No since in the enum we already have a field "displayName" which is the string equivalent of the category